aea/runtime.py

```diff
@@ -261,165 +247,79 @@ def set_loop(self, loop: AbstractEventLoop) -> None:
 
         :param loop: event loop to use.
         """
-        super().set_loop(loop)
-        self.multiplexer.set_loop(self.loop)
-        self.main_loop.set_loop(self.loop)
-        self._async_stop_lock = asyncio.Lock()
-
-    def _start(self) -> None:
-        """
-        Start runtime synchronously.
+        BaseRuntime.set_loop(self, loop)
 
-        Set event loops for multiplexer and agent run loop.
+    async def run(self) -> None:
+        """Start multiplexeor task."""
```

Comment:
```suggestion
        """Start multiplexer task."""
```

Reply:
Also, is it a good docstring? We are starting the runtime, not the multiplexer.

Reply:
nice point